### ISSUE TITLE
Disable PDF rendering of large files

### ISSUE THREAD

**Issue by aochagavia** [isaacs/github#519]
I just sent the following email to support@github.com:

> I have been bitten a couple of times by the javascript pdf rendering (introduced in https://github.com/blog/1974-pdf-viewing). When the pdf files are too big, it lags my browser so much that I have to close the tab (running Chrome on Windows 8.1 with an i3 and 4GB RAM).
>
> The feature I would like to see is disabling pdf rendering for large files, providing a button to activate it in case your computer is powerful enough.

**Comment by zjiayao** [comment]
Exactly. For me, even not-so-large pdf would cause my browser frozen (running Chrome on OSX with an i5 and 4GB RAM).
